Title: 145671 – java-config-1 errors out without a 1.4 JDK
URL: https://bugs.gentoo.org/show_bug.cgi?id=145671

Go to: 
		Gentoo Home
Documentation
Forums
Lists
Bugs
Planet
Store
Wiki
Get Gentoo!





Gentoo's Bugzilla – Bug 145671
java-config-1 errors out without a 1.4 JDK
Last modified: 2006-08-31 09:05:04 UTC node [vulture]


Home
| New–[Ex]
| Browse
| Search
| Privacy Policy

| 





[?]
| Reports

| 
Requests

| 
Help


| 
New Account


| 
Log In





[x]



| 
Forgot Password

Login:




[x]













Bug 145671 
      - java-config-1 errors out without a 1.4 JDK


Summary:
java-config-1 errors out without a 1.4 JDK
    








Status:
    

RESOLVED
          WORKSFORME
      






Alias:


        None
    





Product:

Gentoo Linux




Classification:

Unclassified




Component:

Current packages

  (show other bugs)



Hardware:

All
        Linux
      







Importance:
      
High
       normal
      


Assignee:

Gentoo Linux bug wranglers








URL:







Whiteboard:




Keywords:








Depends on:







Blocks:









 





      Reported:
    
2006-08-30 21:58 UTC by indarios





      Modified:
    
2006-08-31 09:05 UTC
      (History)
    




          CC List:
        

0 
          users
        








See Also:






Package list:







Runtime testing required:

---























      Attachments
    



Add an attachment
        (proposed patch, testcase, etc.)
    








Note
              You need to
              log in
              before you can comment on or make changes to this bug.
            

















Description


indarios





          2006-08-30 21:58:33 UTC
        

Kio ~ # java-config-1
Traceback (most recent call last):
  File "/usr/bin/java-config-1", line 14, in ?
    from java_config import jc_options
  File "/usr/lib64/python2.4/site-packages/java_config/jc_options.py", line 18, in ?
    from jc_util import get_line_from_env_file
  File "/usr/lib64/python2.4/site-packages/java_config/jc_util.py", line 111, in ?
    JAVA_HOME = get_java_home()
  File "/usr/lib64/python2.4/site-packages/java_config/jc_util.py", line 70, in get_java_home
    for f in os.listdir('/etc/env.d/java/'):
OSError: [Errno 2] No such file or directory: '/etc/env.d/java/'

Installing blackdown-jdk fixes this problem.




Comment 1


Jakub Moc (RETIRED)






          2006-08-31 00:51:09 UTC
        

May I ask what are you trying to configure without appropriate JDK version installed?





Comment 2


indarios





          2006-08-31 09:05:04 UTC
        

It complains about having no /etc/env.d/java/ folder without an 1.4 JDK but if you  either install a 1.4 JDK or make the /etc/env.d/java/ folder the command will work just fine. Seems like a bug to me









Format For Printing
 - XML
 - Clone This Bug
 - Clone In The Same 
                        Product
 - Top of page 







Home
| New–[Ex]
| Browse
| Search
| Privacy Policy

| 





[?]
| Reports

| 
Requests

| 
Help


| 
New Account


| 
Log In





[x]



| 
Forgot Password

Login:




[x]

